Sources/AppStoreConnectCLI/Services/AppStoreConnectService.swift

```diff
@@ -704,6 +705,67 @@ class AppStoreConnectService {
             .map(Model.Profile.init)
     }
 
+    func createProfile(
+        name: String,
+        bundleId: String,
+        profileType: ProfileType,
+        certificateSerialNumbers: [String],
+        deviceUDIDs: [String]
+    ) throws -> Model.Profile {
+        let bundleIdResourceId = try ReadBundleIdOperation(
+            options: .init(bundleId: bundleId)
+        )
+        .execute(with: requestor)
+        .await()
+        .id
+
+        let deviceIds = try deviceUDIDs.compactMap {
```

Comment:
This maybe could be a single call rather than multiple.  You can filter devices by multiple UDID I think.